spec/Dynamic-Validator.md

```diff
@@ -65,10 +65,12 @@ The delegated stakes are returned when the account becomes an eligible account o
 ## Election
 The election is a process that elects validators of a term according to the following rule:
 
-1. Select the candidates who deposited **MIN_DEPOSIT** or more.
-2. Pick **MAX_NUM_OF_VALIDATORS** candidates in order of the amount of received delegations and the amount of deposit that the candidate made.
-3. If there are candidates who have tied delegation scores with the dropouts, drop those candidates as well.
-4. Select **MIN_NUM_OF_VALIDATORS** accounts; they become validators.
+1. Calculate the rankings of candidates.
+   * Candidates who receive the most delegation will have the highest ranking.
+   * If there is a tie between them, candidates with the higher index in the `candidates` list will have higher ranking.
```

Comment:
* If there is a tie between them, candidates with the higher index in the `candidates` list will have the higher ranking.